pkg/pebinaryexecutor/pebinary.go

```diff
@@ -190,6 +200,19 @@ func (p *PEBinaryConfig) KubeProxy(args []string) error {
 	}
 
 	args = append(getArgs(extraArgs), args...)
+	var si int
+	for i, arg := range args {
+		if strings.Contains(arg, "source-vip") {
+			si = i
+		}
+	}
```

Comment:
This appears to do the same thing as `argIndex`, should this have been removed?

Reply:
Nice catch, fixed and retested.